Makefile

```diff
@@ -26,7 +26,7 @@ clean: ## remove cache files
 	rm -rf dist
 
 clean-env: ## remove the virtual environment directory
-	pdm venv remove -y $(PROJECT)
```

Comment:
Is this necessary? This change will prevent the recipe from removing global venvs

Reply:
`pdm venv create --force` should only create a local `.venv` file which is handled by `rm -rf .venv`.  Though I just noticed that `pdm venv create -n $(PROJECT)-deploy` creates a global venv which isn't handled. Could we also change this to simply `pdm venv create --force`?